crates/pathfinder/src/cairo/ext_py.rs

```diff
@@ -99,6 +99,17 @@ impl Handle {
                     "Only invoke transactions are supported",
                 ))
             }
+            BroadcastedTransaction::DeployAccount(tx) => {
+                add_transaction::AddTransaction::DeployAccount(add_transaction::DeployAccount {
+                    version: tx.version,
+                    max_fee: tx.max_fee,
+                    signature: tx.signature,
+                    nonce: tx.nonce,
+                    class_hash: tx.class_hash,
+                    contract_address_salt: tx.contract_address_salt,
+                    constructor_calldata: tx.constructor_calldata,
+                })
+            }
```

Comment:
Note: this won't work right now, because the underlying Python code won't be able to estimate deploy account transactions. Still, we should leave this code here for now, and then change this into an error in a separate PR we can revert after adding Python support.

Reply:
Added a small comment about it.